Yosemite/Yosemite/Stores/CardPresentPaymentStore.swift

```diff
@@ -242,26 +242,42 @@ private final class WCPayTokenProvider: CardReaderConfigProvider {
         self.remote = remote
     }
 
-    func fetchToken(completion: @escaping(Result<String, Error>) -> Void) {
+    func fetchToken(completion: @escaping(Result<String, CardReaderConfigError>) -> Void) {
```

Comment:
Do we need a separate error (CardReaderConfigError) apart from the CardReaderServiceError enumeration you've added for bad address above?

Since the CardPresentPaymentStore manages all access to the CardReaderService, it would seem appropriate to leave the function signatures as Error and construct and return a CardReaderService.incompleteStoreAddress in the event the DotcomError has store_address_is_incomplete, but otherwise forward the Error intact from the remote?